Products/PloneBooking/i18n/plonebooking_nl.po

```diff
@@ -1,101 +1,101 @@
-# Translation to Dutch

```

Comment:
Are these line deletions and re-additions an error? If not, what were the relevant changes to this file?